webapp/components/create_post.jsx

```diff
@@ -336,7 +338,8 @@ class CreatePost extends React.Component {
         const tutorialStep = PreferenceStore.getInt(Preferences.TUTORIAL_STEP, UserStore.getCurrentId(), 999);
         this.setState({
             showTutorialTip: tutorialStep === TutorialSteps.POST_POPOVER,
-            ctrlSend: PreferenceStore.getBool(Constants.Preferences.CATEGORY_ADVANCED_SETTINGS, 'send_on_ctrl_enter')
+            ctrlSend: PreferenceStore.getBool(Constants.Preferences.CATEGORY_ADVANCED_SETTINGS, 'send_on_ctrl_enter'),
+            centerTextbox: PreferenceStore.get(Constants.Preferences.CATEGORY_DISPLAY_SETTINGS, Constants.Preferences.CHANNEL_DISPLAY_MODE, Preferences.CHANNEL_DISPLAY_MODE_DEFAULT) === Constants.Preferences.CHANNEL_DISPLAY_MODE_CENTERED
```

Comment:
Huh. I guess I assumed it was not because the one that was already there didn't use it! Our constants file is rather horrific. We should probably fix it so stuff like this isn't necessary.